### PR TITLE
Create FieldList Page (Feature#7)

### LINKED ISSUE #7: Create FieldList page

### PR DESCRIPTION
**Pull request by dtymkiv**

### CODE REVIEW

File: ngfg.client/src/components/Field/FieldList.js [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+    constructor(props) {
+        super(props);
+    }
+
+    state = {
+        'fields': []
+    };
+
+    getData = () => {
+        axios.get('http://ngfg.com:8000/api/v1/fields/', {
```
Comment (IvanKostiv) [inline ngfg.client/src/components/Field/FieldList.js]: URL must be const

File: ngfg.client/src/components/Field/FieldsPage.js [hunk 2]
```diff
@@ -0,0 +1,183 @@
...
+        this.setState({filter});
+    };
+
+    handleSearch = (event) => {
+        this.setState({search: event.target.value});
+    };
+
+    render() {
+        console.log("render state");
+        console.log(this.state);
```
Comment (IvanKostiv) [inline ngfg.client/src/components/Field/FieldsPage.js]: Remove console.log

File: ngfg.client/src/components/Field/FieldList.js (hunk 1 above)
Comment (dtymkiv) [reply ngfg.client/src/components/Field/FieldList.js]: fixed

File: ngfg.client/src/components/Field/FieldsPage.js (hunk 2 above)
Comment (dtymkiv) [reply ngfg.client/src/components/Field/FieldsPage.js]: fixed

**Review by Howrran — DISMISSED** [review]

**Review by AndriiKret — DISMISSED** [review]

**Review by yurdosii — APPROVED** [review]

**Review by Howrran — APPROVED** [review]

**Review by AndriiKret — APPROVED** [review]

**Review by IvanKostiv — APPROVED** [review]

**Comment by dtymkiv** [comment]
Created FieldList page 
(Shared fields todo)

**Pull request merged**